frontend/freebase2wikidata.js

```diff
@@ -260,7 +260,56 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
+  
+ function generateNav() {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1914074/25563816/48e34862-2da4-11e7-8a61-7ba511628b34.png)

Considering the lack of space I opted to leave the button where it was while trying to visually differentiate it.